src/main/java/seedu/planner/logic/commands/module/ModuleRemoveCommand.java

```diff
@@ -62,15 +70,17 @@ public CommandResult execute(Model model) throws CommandException {
         if (!model.hasActiveTimeTable()) {
             throw new CommandException(Messages.MESSAGE_NO_TIMETABLE_ACTIVE);
         }
-
-        // Check if module is present in active timetable
-        if (!model.hasEnrollment(moduleCode)) {
-            throw new CommandException(generateFailureMessage(moduleCode));
+        List<String> messages = new ArrayList<>();
+        for (ModuleCode moduleCode : moduleCodes) {
+            // Check if module is present in active timetable
+            if (!model.hasEnrollment(moduleCode)) {
+                // throw new CommandException(generateFailureMessage(moduleCode));
+                messages.add(generateFailureMessage(moduleCode));
+            }
+            model.removeEnrollment(moduleCode);
+            messages.add(generateSuccessMessage(moduleCode));
         }
-
-        model.removeEnrollment(moduleCode);
-
-        return new CommandResult(generateSuccessMessage(moduleCode));
+        return new CommandResult(String.join("\n", messages));
```

Comment:
I had previously tested my implementation of ModuleRemoveCommand.
Please do not alter my implementation of ModuleAddCommand or ModuleRemoveCommand (or any related classes, such as the corresponding parsers)

Reply:
As spoken, let's test it out and see if it works fine. No point being sceptical when we can improve user experience.